react-validated-input-component/src/index.jsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class Validator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isValid: false,
+      passLength: 0
+    };
+    this.passwordLength = this.passwordLength.bind(this);
+  }
+
+  passwordLength() {
+    this.setState({ passLength: this.state.passLength + 1 });
+    if (this.state.passLength === 7) {
+      this.render();
+    }
+  }
+
+  render() {
+    if (this.state.passLength === 0) {
+      return (
+        <>
+          <form className="flex-form">
+            <label>Password</label>
+            <input type="password" name="user_password" onChange={this.passwordLength}></input>
```

Comment:
Be sure to also pass a `value` prop to your `input` element so that it is fully "controlled".